src/.vuepress/theme/Sidebar.vue

```diff
@@ -41,7 +41,7 @@
                   $page.path.startsWith(item__1.path) ? '2' : ''
                 "
               >
-                {{ item__1.frontmatter.title }}
+                {{ item__1.frontmatter.title.split("|")[0] }}
```

Comment:
What happen if title is not defined ?

Reply:
title can't be defined because it's set as  "required" in frontmatter lint so the project won't run without a title in every page

Reply:
perfect !